robusta_krr/core/integrations/prometheus/loader.py

```diff
@@ -46,26 +46,40 @@ def __init__(self, *, cluster: Optional[str] = None) -> None:
         )
         loader = self.get_metrics_service(api_client=self.api_client, cluster=cluster)
         if loader is None:
-            raise PrometheusNotFound("No Prometheus or metrics service found")
+            raise PrometheusNotFound(
+                f"Wasn't able to connect to any Prometheus service in {cluster or 'inner'} cluster\n"
+                "Try using port-forwarding and/or setting the url manually (using the -p flag.).\n"
+                "For more information, see 'Giving the Explicit Prometheus URL' at https://github.com/robusta-dev/krr?tab=readme-ov-file#usage"
+            )
 
         self.loader = loader
 
-        logger.info(f"{self.loader.name} connected successfully for {cluster or 'default'} cluster")
+        logger.info(f"{self.loader.name()} connected successfully for {cluster or 'default'} cluster")
 
     def get_metrics_service(
         self,
         api_client: Optional[ApiClient] = None,
         cluster: Optional[str] = None,
     ) -> Optional[PrometheusMetricsService]:
-        for service_name, metric_service_class in METRICS_SERVICES.items():
+        if settings.prometheus_url is not None:
+            logger.info("Prometheus URL is specified, will not auto-detect a metrics service")
+            metrics_to_check = [PrometheusMetricsService]
+        else:
+            logger.info("No Prometheus URL is specified, trying to auto-detect a metrics service")
+            metrics_to_check = [VictoriaMetricsService, ThanosMetricsService, PrometheusMetricsService]
```

Comment:
could this be 
maybe change `METRICS_SERVICES` to
`METRICS_SERVICES = [VictoriaMetricsService, ThanosMetricsService, PrometheusMetricsService] `
So we can use it here or remove `METRICS_SERVICES` all together